.github/workflows/doc_check.yml

```diff
@@ -0,0 +1,42 @@
+name: Documentation Check for Magick++
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+  schedule:
+    - cron: 0 6 * * *
+
+jobs:
+  doxygen-check:
+    name: 'Documentation Check'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository content
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.13'
+
+      - name: Install Python dependencies
+        run: python -m pip install --user pyarrow beautifulsoup4 pandas lxml
+
+      - name: Install Doxygen
+        run: sudo apt-get update && sudo apt-get install -y doxygen
+
+      - name: Compare documentation
+        run: |          
+          # first generate the Magick++ documentation to be up to date with the current code
+          cd config && doxygen Magick++.dox.in &> /dev/null
+          cd ..
+          
+          # documentation files with function signatures that we can compare with their corresponding doxygen file (currently)
+          DOC_HTML=("Blob.html" "CoderInfo.html" "Geometry.html" "Image++.html" "Montage.html" "Pixels.html")
+          DOXY_HTML=("Blob.html" "CoderInfo.html" "Geometry.html" "Image.html" "Montage.html" "Pixels.html")  # Image.html is the only one that differs from the doxygen file
+          for i in "${!DOC_HTML[@]}"; do
+            python3 scripts/compare_signatures.py www/Magick++/"${DOC_HTML[$i]}" www/api/Magick++/classMagick_1_1"${DOXY_HTML[$i]}"

```

Comment:
I would like the build to fail when this script finds an issue in one of the files. But only after each file has been tested. And this doesn't seem to be happening now.